sdk/monitor/Azure.Monitor.Ingestion/tests/BatchingTest.cs

```diff
@@ -46,7 +46,7 @@ public void ValidateBatchingMultiChunkNoGzip()
             }
             IEnumerable<LogsIngestionClient.BatchedLogs<IEnumerable>> x = LogsIngestionClient.Batch(entries);
             Assert.Greater(x.Count(), 1); //ideally should be 2 batches
-            Assert.Less(x.Count(), 5);
+            Assert.Less(x.Count(), 3);
```

Comment:
Calling Count method will cause this to enumerate each time. Instead we can do a ToList and use the Count property.